apps/design/frontend/src/clone_election_button.test.tsx

```diff
@@ -62,54 +54,48 @@ afterEach(() => {
 });
 
 test('clones immediately for non-Vx users', async () => {
-  mockUseUserFeatures.mockReturnValue(userFeatureConfigs.nh);
-
-  const { election } = generalElectionRecord(nonVxUser.orgId);
-  const { history } = renderButton(
-    nonVxUser,
-    <CloneElectionButton election={election} />
-  );
+  mockUserFeatures(apiMock, user, { ACCESS_ALL_ORGS: false });
+  const { election } = generalElectionRecord(user.orgId);
+  const { history } = renderButton(<CloneElectionButton election={election} />);
 
   const newElectionId = 'new-election' as ElectionId;
   mockGenerateId.mockReturnValue(newElectionId);
   apiMock.cloneElection
     .expectCallWith({
       destId: newElectionId,
-      destOrgId: nonVxUser.orgId,
+      destOrgId: user.orgId,
       srcId: election.id,
-      user: nonVxUser,
+      user,
     })
     .resolves(newElectionId);
 
-  userEvent.click(screen.getButton(`Make a copy of ${election.title}`));
+  userEvent.click(await screen.findButton(`Make a copy of ${election.title}`));
   await sleep(0); // Allow redirect to resolve
   expect(history.location.pathname).toEqual(`/elections/${newElectionId}`);
 });
 
 const VX_ORG = {
-  id: vxUser.orgId,
+  id: user.orgId,
   name: 'votingworks',
   displayName: 'VotingWorks',
 } as const;
 
 const NON_VX_ORG = {
-  id: nonVxUser.orgId,
+  id: user.orgId,
   name: 'not-voting-works',
   displayName: 'Not VotingWorks',
 } as const;
 
 test('shows org picker for Vx users', async () => {
```

Comment:
The test names might now be inconsistent with behavior. Is the user's org now the thing that determines whether the org picker is shown? Or is it the `ACCESS_ALL_ORGS` flag response from the backend?